### PR TITLE
Prepare v0.8.3

### PR DESCRIPTION
**Pull request by Zeitsperre**
## Overview

Changes:

* Added some workflows for automated publishing
* Bumped the version number (removed previous tag with version 0.8.3)
* Reduced the number of redundant builds for PRs

## Additional Information

Links to other issues or sources.

### CODE REVIEW

**Review by huard — APPROVED** [review]

**Comment by huard** [comment]
@cehbrecht Ok with automating the publishing to PyPI ?

**Review by cehbrecht — APPROVED** [review]

**Comment by cehbrecht** [comment]
> @cehbrecht Ok with automating the publishing to PyPI ?

@Zeitsperre @huard I'm fine with automatic publishing to pypi 👍 Maybe add some short hints in the `README.rst` how the release procedure goes?

File: docs/source/development.rst [hunk 1]
```diff
@@ -63,3 +63,17 @@ Make a new version of Birdy in the following steps:
 See the bumpversion_ documentation for details.
 
 .. _bumpversion: https://pypi.org/project/bumpversion/
+
+Release a new version
+=====================
+
+Leveraging GitHub Workflows, maintainers can release new versions of Birdy automatically:
+
+* Within the repository secrets, set two API tokens (`PYPI_API_TOKEN` and `TEST_PYPI_API_TOKEN`) with push access to Birdy on PyPI and TestPyPI, respectively.
```
Comment (huard) [inline docs/source/development.rst:72]: I'd remove that since it does not have to be done again for the next releases.

**Review by huard — APPROVED** [review]

File: docs/source/development.rst (hunk 1 above)
Comment (Zeitsperre) [reply docs/source/development.rst:72]: Got it. My API Tokens will remain valid indefinitely, but if ever they are revoked, the action will let us know.

**Pull request merged**